src/components/TableProperties.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div class="vue-ads-relative vue-ads-z-50">
+        <div class="vue-ads-properties-card vue-ads-mt-3 vue-ads-absolute vue-ads-shadow-2xl" :class="contentClasses">
+            <span>Properties</span>
+            <div class="vue-ads-ml-5 vue-ads-mt-2 vue-ads-mr-2">
+                <div class="vue-ads-mt-1 vue-ads-flex" v-for="column of transformedColumns" :key="column.property">
+                    <input
+                        type="checkbox"
+                        :id="column.property"
+                        :value="column.property"
+                        v-model="selectedColumns"
+                        :disabled="selectedColumns.length === 1 && selectedColumns[0] === column.property"
```

Comment:
I would suggest move these long conditions to the computed property

Reply:
If we paste this code into computed property, this computerized property will be re-calculated each time an element is selected., thus caching will not work, which is not very good. So computed property is redundant here.

Reply:
We need code review for code style assessment and not for functional testing. If you worry about caching in computed properties, you can use `cache: false` or a `method` for that. Anyway, don't ask for code review if you don't wan't to implement suggestions :)

Reply:
JFYI: https://v2.vuejs.org/v2/style-guide/?redirect=true#Simple-expressions-in-templates-strongly-recommended